### PR TITLE
Inject CMP version from project version

### PR DESCRIPTION
**Pull request by rafamarquezv**
### Inject CMP version from project version

Set **cmpVersion** automatically using the first digit from the **package.json** version on build time:

- remove cmpVersion from internals configuration file
- receive cmpVersion from bootstrap
- update readme
- fix tests
- include a webpack config file and use it to extract the major digit from package.json version
- Simplify NPM tasks

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
+/* eslint-disable no-undef */
 import initializeCMP from './cmp/infrastructure/cmpInitializer'
 import createEvent from './cmp/infrastructure/createEvent'
 import registerWindowCMP from './cmp/infrastructure/controller/windowCommunicationRegistry'
 import registerIframeCommunication from './cmp/infrastructure/controller/iframeCommunicationRegistry'
 
-initializeCMP({configuration: window.__cmp_config})
+initializeCMP({
+  configuration: window.__cmp_config,
+  cmpVersion: CMP_VERSION
```
Comment (victuxbb) [inline src/index.js]: nice!

File: webpack.config.babel.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import webpack from 'webpack'
+import path from 'path'
+import CleanWebpackPlugin from 'clean-webpack-plugin'
+
+const nodeEnv = process.env.NODE_ENV || 'dev'
+
+const getMajorVersionFromPackageJsonVersion = () => {
```
Comment (victuxbb) [inline webpack.config.babel.js]: cool

File: package.json [hunk 3]
```diff
@@ -1,17 +1,16 @@
 {
   "name": "@schibstedspain/cmp",
-  "version": "0.0.1",
+  "version": "1.0.1",
```
[the remark this answers is not in the document]
Comment (rafamarquezv) [reply package.json]: @victuxbb you are right, fixed.

**Comment by rafamarquezv** [comment]
@victuxbb you are right, fixed.

File: webpack.config.babel.js [hunk 4]
```diff
@@ -2,15 +2,20 @@ import webpack from 'webpack'
 import path from 'path'
 import CleanWebpackPlugin from 'clean-webpack-plugin'
 
+const WEBPACK_MODE_PRODUCTION = 'production'
```
Comment (victuxbb) [inline webpack.config.babel.js]: Love it!

**Pull request merged**